ios/build.sh

```diff
@@ -124,51 +121,28 @@ update_bundle ( ) {
 
         cd $base_dir
     fi
-
-    #Copy the updated bundle to our output folder.
-    cp -Rf $KMEI_RESOURCES/Keyman.bundle ${KMEI_OUTPUT_FOLDER}
 }
 
 # First things first - update our dependencies.
-
-if ! [ -d "${KMEI_OUTPUT_FOLDER}" ]; then
-    mkdir -p "${KMEI_OUTPUT_FOLDER}"
-fi
-
 update_bundle
 
 echo
 echo "Building KMEI..."
 
 #OTHER_CFLAGS=-fembed-bitcode is relied upon for building the samples by command-line.  They build fine within XCode itself without it, though.
 
-rm -r $KMEI_BUILD_PATH/${CONFIG}-iphoneos 2>/dev/null
-xcodebuild -quiet -project engine/KMEI/KeymanEngine.xcodeproj -target KME-iphoneos OTHER_CFLAGS=-fembed-bitcode \
-  -configuration $CONFIG
-assertFileExists $KMEI_BUILD_PATH/${CONFIG}-iphoneos/libKME-iphoneos.a
-
-rm -r $KMEI_BUILD_PATH/${CONFIG}-iphonesimulator 2>/dev/null
-xcodebuild -quiet -project engine/KMEI/KeymanEngine.xcodeproj -sdk iphonesimulator PLATFORM_NAME=iphonesimulator \
-  -target KME-iphonesimulator OTHER_CFLAGS=-fembed-bitcode -configuration $CONFIG
-assertFileExists $KMEI_BUILD_PATH/${CONFIG}-iphonesimulator/libKME-iphonesimulator.a
+rm -r $KMEI_BUILD_PATH/$CONFIG-universal 2>/dev/null
+BUILD_FRAMEWORK_PATH=$KMEI_BUILD_PATH/$CONFIG-universal/KeymanEngine.framework
+xcodebuild -quiet -project engine/KMEI/KeymanEngine.xcodeproj -target KME-universal -configuration $CONFIG \
+  $CODE_SIGN_IDENTITY $CODE_SIGNING_REQUIRED $DEV_TEAM
+assertDirExists "$BUILD_FRAMEWORK_PATH"
 
-# Combine the two builds into KMEI.
-rm -f ${KMEI_OUTPUT_FOLDER}/libKeyman.a  2>/dev/null
-
-lipo -create "$KMEI_BUILD_PATH/${CONFIG}-iphonesimulator/libKME-iphonesimulator.a" "$KMEI_BUILD_PATH/${CONFIG}-iphoneos/libKME-iphoneos.a" -output "${KMEI_OUTPUT_FOLDER}/libKeyman.a"
```

Comment:
I assume that using a 'framework' means we no longer need to create that 'universal library' thing?

Reply:
There's no need to create `libKeyman.a` as a universal library. However `lipo` is still used to create a universal binary within the framework. However that's now done in the KME-universal target in the Xcode project.

Reply:
Is that universal binary used by the framework?  If so, we probably need the old schemes, since I'm almost certain KME-universal relies upon those two as prerequisites.  (Try doing a 'make -clean' and then the standard build.)

Reply:
An interesting note - The KeymanEngine.framework build product does not appear to include the Swift libraries, unlike the app's produced archive and .ipa files.  I wonder if we're missing something significant during the archiving step.